File: Java/T3/codigos1920/proyectos/T03_Spinner/app/src/main/java/com/borja/t03_spinner/utils/Persona.java
package com.borja.t03_spinner.utils;

import androidx.annotation.NonNull;

public class Persona {

    String nombre, apellido;
    int edad;
    boolean disponiblidad;

    public Persona(String nombre, String apellido, int edad, boolean disponiblidad) {
        this.nombre = nombre;
        this.apellido = apellido;
        this.edad = edad;
        this.disponiblidad = disponiblidad;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }

    public int getEdad() {
        return edad;
    }

    public void setEdad(int edad) {
        this.edad = edad;
    }

    public boolean isDisponiblidad() {
        return disponiblidad;
    }

    public void setDisponiblidad(boolean disponiblidad) {
        this.disponiblidad = disponiblidad;
    }

    @NonNull
    @Override
    public String toString() {
        return getNombre();
    }
}

File: Kotlin/T1/codigos2223/Inicio/src/main/kotlin/herencia/Persona.kt
package herencia

open class Persona (var nombre: String, var apellido: String, val dni: String) {

    // nombre, apellido, dni
    var edad: Int=0;

    constructor(nombre: String, apellido: String, dni: String, edad: Int):
            this(nombre,apellido,dni){
        this.edad = edad;
    }

    open fun mostrarDatos(){
        println("Nombre: $nombre")
        println("Apellido: $apellido")
        println("DNI: $dni")
        println("Edad: $edad")
    }

}